plat/kvm/Makefile.uk

```diff
@@ -25,9 +25,9 @@ LIBKVMPLAT_CINCLUDES-y         += -I$(LIBKVMPLAT_BASE)/include
 LIBKVMPLAT_CINCLUDES-y         += -I$(UK_PLAT_COMMON_BASE)/include
 LIBKVMPLAT_CINCLUDES-y         += -I$(UK_PLAT_DRIVERS_BASE)/include
 
-LIBKVMPLAT_ASFLAGS             += -DKVMPLAT
-LIBKVMPLAT_CFLAGS              += -DKVMPLAT
-LIBKVMPLAT_CXXFLAGS            += -DKVMPLAT
+LIBKVMPLAT_ASFLAGS             += -DKVMPLAT -fno-sanitize=kernel-address
```

Comment:
Maybe a global platform flags variable is needed.

Reply:
When KASAN is enabled globally, we don't want the platform code to be instrumented (wouldn't make sense since kasan is initialized after the allocator).